recipe/meta.yaml

```
@@ -54,6 +55,7 @@ requirements:
     - m2-m4  # [win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - ninja  # [win]
```

```suggestion
    - ninja-base  # [win]
```
`ninja-base` comes without an additional `python`